### PR TITLE
Two small modifications

### PR DESCRIPTION
**Pull request by Falsejoey**
getUserOwnedGames and Game.js

### CODE REVIEW

File: src/structures/Game.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
+const cdn = 'https://steamcdn-a.akamaihd.net/steamcommunity/public/images/apps';
+
 class Game {
 	constructor(game) {
+		this.name = game.name;
 		this.appID = game.appid;
 		this.playTime = game.playtime_forever;
-		this.playTime2 = game.playtime_2weeks || 0;
```
Comment (xDimGG) [inline src/structures/Game.js]: Why was this removed?

Comment (Falsejoey) [reply src/structures/Game.js]: Sorry about that It should have been added back in, originally i thought it was absent from the api after adding the `include_appinfo` tag, because I tested a steam id with no playtime in the last 2 weeks. Ill add it back right now
